tests/test_enterprise/api/test_views.py

```diff
@@ -32,24 +32,32 @@
 CATALOGS_LIST_ENDPOINT = reverse('catalogs-list')
 CATALOGS_DETAIL_ENDPOINT = reverse('catalogs-detail', (1, ))
 CATALOGS_COURSES_ENDPOINT = reverse('catalogs-courses', (1, ))
-ENTERPRISE_CATALOGS_LIST_ENDPOINT = reverse('enterprise-catalogs-list')
+ENTERPRISE_CATALOGS_LIST_ENDPOINT = reverse('enterprise_catalogs-list')
 ENTERPRISE_CATALOGS_DETAIL_ENDPOINT = reverse(
-    'enterprise-catalogs-detail',
+    'enterprise_catalogs-detail',
+    kwargs={'pk': FAKE_UUIDS[1]}
+)
+ENTERPRISE_CATALOGS_CONTAINS_CONTENT_ENDPOINT = reverse(
+    'enterprise_catalogs-contains-content',
     kwargs={'pk': FAKE_UUIDS[1]}
 )
 ENTERPRISE_CATALOGS_COURSE_RUN_ENDPOINT = reverse(
     # pylint: disable=anomalous-backslash-in-string
-    'enterprise-catalogs-course-runs/(?P<course-id>[^/+]+(/|\+)[^/+]+(/|\+)[^/?]+)',
+    'enterprise_catalogs-course-runs/(?P<course-id>[^/+]+(/|\+)[^/+]+(/|\+)[^/?]+)',
     kwargs={'pk': FAKE_UUIDS[1], 'course_id': TEST_COURSE}
 )
 ENTERPRISE_CATALOGS_PROGRAM_ENDPOINT = reverse(
-    'enterprise-catalogs-programs/(?P<program-uuid>[^/]+)',
+    'enterprise_catalogs-programs/(?P<program-uuid>[^/]+)',
     kwargs={'pk': FAKE_UUIDS[1], 'program_uuid': FAKE_UUIDS[3]}
 )
 ENTERPRISE_COURSE_ENROLLMENT_LIST_ENDPOINT = reverse('enterprise-course-enrollment-list')
 ENTERPRISE_CUSTOMER_COURSES_ENDPOINT = reverse('enterprise-customer-courses', (FAKE_UUIDS[0],))
 ENTERPRISE_CUSTOMER_ENTITLEMENT_LIST_ENDPOINT = reverse('enterprise-customer-entitlement-list')
 ENTERPRISE_CUSTOMER_LIST_ENDPOINT = reverse('enterprise-customer-list')
+ENTERPRISE_CUSTOMER_CONTAINS_CONTENT_ENDPOINT = reverse(
+    'enterprise-customer-contains-content',
```

Comment:
@douglashall Should this be `enterprise_customer-contains-content` to match the naming scheme of the remaining endpoints? Mixing underscores and dashes in endpoint names is a little weird, but I'm assuming it has something to do with [DRF default paths](https://openedx.atlassian.net/browse/ENT-668)?

Reply:
Yeah I suppose that's something for ENT-668 to take care of. It wouldn't be reasonable to do it as part of this ticket since that'd require changes across the ecosystem to be safe, I believe.

Reply:
I think DRF likes to use dashes in URL names by default. I agree that we should follow up on this and make things consistent when we work on ENT-688. I'm going to revert the `enterprise_catalogs` URL names above in the meantime to keep them consistent with DRF naming.